### DISCUSSION TITLE
unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by mainisameer** [socketio/socket.io#4871] [Q&A]
We have a desktop application that connects to our server using socket.io on some user desktops we are getting the following error message unable to get local issuer certificate. The application is written in electron, any guidance on this would be much appreciated

**Comment by darrachequesne** [comment]
Hi! Are you behind a proxy? (like https://github.com/electron/electron/issues/9424)

See also: https://github.com/search?q=repo%3Aelectron%2Felectron+%22unable+to+get+local+issuer+certificate%22&type=issues

**Reply by mainisameer** [reply]
Our is different as we are not installing NPM package on the desktop but our distribution package. And the application is not able to connect to the server
